Dixit_Logic/Interfaces/IImageAccess.cs

```diff
@@ -10,7 +10,7 @@ namespace Dixit_Logic.Interfaces
     /// <summary>
     /// This interface loads the images for the cards.
     /// </summary>
-    interface IImageAccess
+    public interface IImageAccess
```

Comment:
@ballaibrigitt42 This is not sufficient. Please change the methods to public as well